velox/common/memory/MappedMemory.cpp

```diff
@@ -338,6 +338,64 @@ std::shared_ptr<MappedMemory> MappedMemory::addChild(
   return std::make_shared<ScopedMappedMemory>(this, tracker);
 }
 
+namespace {
+// Returns the size class size that corresponds to 'size'.
+int32_t sizeClassSize(
```

Comment:
Should the return type be MachinePageCount to be more readable?
The helper method name could be roundUpToSizeClassSize